packages/app/src/cli/services/app-logs/utils.ts

```diff
@@ -117,14 +117,16 @@ const generateFetchAppLogUrl = async (
   return url
 }
 
-export const fetchAppLogs = async (
-  jwtToken: string,
-  cursor?: string,
+export interface FetchAppLogsOptions {
+  jwtToken: string
+  cursor?: string
   filters?: {
     status?: string
     source?: string
-  },
-): Promise<Response> => {
+  }
+}
+
+export const fetchAppLogs = async ({jwtToken, cursor, filters}: FetchAppLogsOptions): Promise<Response> => {
```

Comment:
I'm wondering if this should be moved into the partners client. Then you know its always called via the client.